github.js

```diff
@@ -35,16 +34,18 @@ async function constructPrMsg(sg, branch) {
  */
 function constructTrainNavigation(branchToPrDict, currentBranch, combinedBranch) {
   let contents = '<pr-train-toc>\n\n';
-  let tableData = [['', 'PR', 'Description']];
+  let prList = [];
   Object.keys(branchToPrDict).forEach((branch) => {
-    const maybeHandRight = branch === currentBranch ? '👉 ' : ' ';
+    const maybeHandRight = branch === currentBranch ? ' 👈' : ' ';
     const combinedInfo = branch === combinedBranch ? ' **[combined branch]** ' : ' ';
-    const prTitle = branchToPrDict[branch].title.trim();
     const prNumber = `#${branchToPrDict[branch].pr}`;
-    const prInfo = `${combinedInfo}${prTitle}`.trim();
-    tableData.push([maybeHandRight, prNumber, prInfo]);
+    const prInfoHtml = `1. ${prNumber}${combinedInfo}${maybeHandRight}`
+    prList.push(prInfoHtml);
   });
-  contents += table(tableData, { stringLength: width }) + '\n';
+  if (prList.length > 0) {
```

Comment:
**Nit**: OOC, can this ever be empty? (I would think that there is always at least one branch when one runs pr-train, no?)